txtdirect.go

```diff
@@ -33,7 +33,9 @@ const (
 	basezone        = "_redirect"
 	defaultSub      = "www"
 	defaultProtocol = "https"
+	keepalive       = 30
 	logFormat       = "02/Jan/2006:15:04:05 -0700"
+	timeout         = 30 * time.Second
```

Comment:
let's call them proxyTimeout and proxyKeepalive or else it's harder to distinguish.

Reply:
Sounds good 👍